config/routes.rb

```diff
@@ -1,4 +1,9 @@
 Rails.application.routes.draw do
-  get '/', to: 'application#welcome'
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+  get '/', to: 'application#welcome'
+
+  resources :merchants, only: [:index] do
+    resources :items #, only: [:index, :show, :edit, :update]
```

Comment:
I think it would be good to explicitly indicate what actions to generate routes for with the `only`.  I say this because I don't think we have a `destroy` action.  Another option might be to use `exclude` instead of `only` and list the actions you **don't** want.

Reply:
True true. Updated using exlude for destroy.